x/bank/module.go

```diff
@@ -133,9 +135,12 @@ func (am AppModule) LegacyQuerierHandler(legacyQuerierCdc *codec.LegacyAmino) sd
 // InitGenesis performs genesis initialization for the bank module. It returns
 // no validator updates.
 func (am AppModule) InitGenesis(ctx sdk.Context, cdc codec.JSONMarshaler, data json.RawMessage) []abci.ValidatorUpdate {
+	start := time.Now()
 	var genesisState types.GenesisState
-
 	cdc.MustUnmarshalJSON(data, &genesisState)
+	telemetry.MeasureSince(start, "InitGenesis", "crisis", "unmarshal")
+	fmt.Println("BANK: UnmarshalJSON", time.Since(start))
```

Comment:
This would print a message to `/dev/stdout`. Errors and diagnostics should go to `/dev/stderr` - as per POSIX's recommendations. Is it really necessary to print this message at all? If so, how about redirecting it to the module's logger?

```suggestion
	fmt.Fprintln(os.Stderr, "BANK: UnmarshalJSON", time.Since(start))
```

Reply:
I know, I was about to say to not look at it. I will remove once everything is confirmed.

Reply:
Splendid, thanks